project/plugins.sbt

```diff
@@ -5,8 +5,8 @@ addSbtPlugin("org.scalameta" % "sbt-scalafmt" % "2.4.3")
 addSbtPlugin("ch.epfl.scala" % "sbt-scalafix" % "0.9.29")
 addSbtPlugin("org.portable-scala" % "sbt-scalajs-crossproject" % "1.1.0")
 addSbtPlugin("org.portable-scala" % "sbt-scala-native-crossproject" % "1.1.0")
-addSbtPlugin("org.scala-native" % "sbt-scala-native" % "0.4.0")
-addSbtPlugin("org.scala-js" % "sbt-scalajs" % "1.6.0")
+addSbtPlugin("org.scala-native" % "sbt-scala-native" % "0.4.4")
+addSbtPlugin("org.scala-js" % "sbt-scalajs" % "1.10.0")
```

Comment:
Scala.js 1.10 is forward+backward compatible through Scala.js 1.8. So downstreams can stay on 1.8 if they wish and are not forced to upgrade.